project/.travis.yml.twig

```diff
@@ -55,7 +59,9 @@ before_install:
   - if [ "$TARGET" = 'test' ]; then composer self-update --stable; fi;
   - if [ "$TARGET" = 'test' ]; then composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN; fi;
   - if [ "$TARGET" = 'test' ]; then sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json; fi;
-  - if [ "$TARGET" = 'test' -a "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+{% for package_name,package_versions in versions if package_versions|length > 0 %}
+  - if [ "$TARGET" = 'test' -a "${{ package_name|upper }}" != "" ]; then composer require "{{ packages[package_name] }}:${{ package_name|upper }}" --no-update; fi;
```

Comment:
@OskarStark I think the `packagen_name|upper` is the reason

Reply:
Jepp, that seems to be correct, but is not needed anymore with GithubActions

Reply:
For reference:
https://github.com/sonata-project/SonataAdminBundle/blob/7ca2b6e800873159d3ba23bdff613be096dac4b4/.github/workflows/test.yaml#L22-L47